docs/ossm/injection/README.md

```diff
@@ -0,0 +1,105 @@
+## Installing the Sidecar
+### Injection
+In order to take advantage of all of Istio’s features, pods in the mesh must be running an Istio sidecar proxy.
+
+The following sections describe automatic Istio sidecar injection in the pod’s namespace. Manual injection using the `istioctl` command is not supported by OpenShift Service Mesh.
```

Comment:
One thing I don't understand is that this intro talks about "injection in the pod’s namespace", but below it documents that you can add a label to a pod alone if you only want the individual pod to be injected... is that correct? 

I am also not sure about the name "Automatic vs manual"  injection relates to labels vs istioctl. Perhaps we should just talk about "sidecar injection" and describe the method with labels where you have two option: namespace level or pod level. Then mention at the bottom that the `istioctl kube-inject` command is not supported.

I realize a lot of this is taken from upstream injection documentation.

Reply:
Yes, it's correct. You can add a label either to a namespace or a pod. Both will cause that the pod will be injected.

I will try to reword this so it's clearer.